super_csv/csv_processor.py

```diff
@@ -154,14 +154,14 @@ def get_iterator(self, rows=None, columns=None, error_data=False):
         """
         Export the CSV as an iterator.
```

Comment:
I know you didn't touch this, but this docstring is basically completely useless. 

Can you add some info in here about the parameters